source/blog/2017-11-20-ember-2-17-released.md

```diff
@@ -0,0 +1,201 @@
+[Ember.js 2.17.0 release page](https://github.com/emberjs/ember.js/releases/tag/v2.17.0).
+
+### Upcoming Changes in Ember.js 2.18
+
+Ember.js 2.18 will introduce two new features:
+
+* TODO
+* TODO
+
+#### Deprecations in Ember.js 2.18
+
+Two new deprecations are introduces in Ember.js 2.18:
+
+* TODO
+* TODO
+
+For more details on the upcoming changes in Ember.js 2.18, please review the
+[Ember.js 2.18.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v2.18.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.17
+
+#### Deprecations in Ember Data 2.17
+
+Two new deprecations are introduces in Ember Data 2.17:
+
+* TODO
+* TODO
+
+For more details on changes in Ember Data 2.17, please review the
+[Ember Data 2.17.0 release page](https://github.com/emberjs/data/releases/tag/v2.17.0).
+
+### Upcoming changes in Ember Data 2.18
+
+* [#6955](https://github.com/ember-cli/ember-cli/pull/6955) Discover
+  dependencies of npm-linked addons [@ef4](https://github.com/ef4)
+
+* [#7419](https://github.com/ember-cli/ember-cli/pull/7419) Delete
+  crossdomain.xml [@sandstrom](https://github.com/sandstrom)
+
+* [#7406](https://github.com/ember-cli/ember-cli/pull/7406) Remove livereload
+  url from output [@topaxi](https://github.com/topaxi)
+
+#### Deprecations in Ember Data 2.18
+
+For more details on the upcoming changes in Ember Data 2.18, please review the
+[Ember Data 2.18.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v2.18.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
+your projects using `yarn` run:
+
+```
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```
+npm install --save-dev ember-cli
+```
+
+After running the upgrade command run `ember init` inside of the project
+directory to apply the blueprint changes. You can preview those changes for
+[applications](https://github.com/ember-cli/ember-new-output/compare/v2.15.0...v2.17.0)
+and
+[addons](https://github.com/ember-cli/ember-addon-output/compare/v2.15.0...v2.17.0).
+
+### Changes in Ember CLI 2.17
+
+Ember CLI previously displayed a warning when you removed `ember-cli-shims`, but
+the dependency is no longer needed if none of your dependencies are using the
+old shims and all of them are using `ember-cli-babel` 6.6.0 or above.
+
+With the release of Ember 2.17, Ember 2.16 is an LTS candidate. To ensure that
+the default `ember-try` configurations are up to date for new applications,
+Ember 2.8 LTS was retired from the configuration and Ember 2.16 was added.
+
+Continuing the trend towards headless browsers, `ember test` in Ember CLI 2.17
+will now run the tests in headless Chrome. To achieve the previous behaviour,
+use `ember test --server`.
```

Comment:
We have been using headless Chrome for a while already. The change here was to *not* use headless when running `ember test --server`.